docs/db/migrations.md

```diff
@@ -154,6 +154,44 @@ type Note {
 
```

Comment:
Would it be possible to move that docs from db. Contents here focusing on building schema relationships and many people struggling to find it.

Reply:
So, remove it from database section completely? It is use for migrations though.

Reply:
hmm.. That is actually good question. I see this as way to design schema. 

Then db migrations can literally reference this information explaining how db is generated for individual use cases etc. We are missing that at the moment

Reply:
Yep I considered this but then there are variations between the two. For example:

- `@db.name` annotation is used in migrations, but will do nothing in data model, i.e. we do not auto-map so it is misleading to put it there.
- Migrations require the `id: ID` field, but Graphback does not require this in runtime or generation, just migrations does.

Reply:
Ah.. let me think about it then